### ISSUE TITLE
Error message trying to add integration

### ISSUE THREAD

**Issue by OtmShank751** [jeatheak/Mitsubishi-WF-RAC-Integration#84]
Hi there,

I am trying to add this integration to Home Assistant on Rpi4. I tried to add through HACS and it would not add. 

I was able to manually add the folder to the custom components folder via SAMBA. 

It now appears in my list of available integrations, but when I try to add the integration I get the message below. Can you help?

Thanks

Sam

![mitsracerror](https://github.com/jeatheak/Mitsubishi-WF-RAC-Integration/assets/159682905/6a76a2ec-2e17-43c6-b6fb-7f77b21c98e4)

**Comment by jeatheak** [comment]
Do you have a more detailed error log?
and which versions are running? (core, os)

**Comment by OtmShank751** [comment]
Sorry, I am unable to locate the log file for the failed install. I searched the core logs but they don't go back far enough.

What I can say is that whatever I tried, it failed to add the integration and would return the error above. So I left it at that and forgot about it.

THEN, after upgrading Home Assistant Core (2024.2.2) and Supervisor (2024.01.1), the integration added itself (without me doing anything) and Home Assistant auto detected my AC, so it is now working.

Thank you for getting back to me, if you want to look at this further and can point me in the right direction to find the logs for that day, let me know.

Otherwise, happy to close this issue, thank you for creating this integration!
